plugins/SitesManager/templates/sites-list/site-fields.html

```diff
@@ -31,7 +31,9 @@
             <ul>
                 <li>
                     <span class="title">{{ 'SitesManager_Urls'|translate }}</span>:
-                    {{ site.alias_urls.join(', ') }}
+                    <div = ng-repeat = "url in site.alias_urls">
```

Comment:
There is an unneeded `=` sign. Also can you maybe remove the whitespace before and after the `=` here and below after `href`?